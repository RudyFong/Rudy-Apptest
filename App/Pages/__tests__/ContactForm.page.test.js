import React from 'react';
import renderer from 'react-test-renderer';
import ContactFormPage from '../ContactForm.page';
import {Provider} from 'react-redux';
import {createStore} from 'redux';

jest.mock('../ContactForm.page');
const store = createStore(() => ({}));

describe('ContactForm : ContactForm', () => {
  it('renders correctly', () => {
    const tree = renderer.create(
      <Provider store={store}>
        <ContactFormPage/>
      </Provider>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
