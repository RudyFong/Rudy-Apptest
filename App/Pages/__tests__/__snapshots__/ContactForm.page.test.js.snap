// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactForm : ContactForm renders correctly 1`] = `null`;
